src/main/java/com/labsynch/labseer/chemclasses/rdkit/BBChemStructureService.java

```diff
@@ -80,7 +80,8 @@ public void populateDescriptors(RDKitStructure rdKitStructure) {
 		// canonical	: if false, no attempt will be made to canonicalize the SMILES
 		// allBondsExplicit	: if true, symbols will be included for all bonds.
 		// allHsExplicit	: if true, hydrogen counts will be provided for every atom.
-		rdKitStructure.setSmiles(RDKFuncs.MolToSmiles(mol, true, false, -1, true, false, false, false));
+		// doRandom : undocumented as of 2021-11-29 but default is false
+		rdKitStructure.setSmiles(RDKFuncs.MolToSmiles(mol, true, false, -1, true, false, false, false, false));
```

Comment:
Confused by the change on this line. Before your change, I this `MolToSmiles` had "mol + 7 arguments" but your docs had "mol + 6 arguments".
But you have now added an additional `false` to `MolToSmiles`, so your code has "mol + 8 arguments" and your docs have "mol + 7 arguments".

I think you only needed to add the comment and didn't need to change the line of code.